### ISSUE TITLE
pdd treats a variable that contains TODO word as a marker

### ISSUE THREAD

**Issue by php-coder** [cqfn/pdd#226]
Could we adjust parsing rules to not produce a false positive for the variables that contains `TODO`? At least, if there is no a space or a colon after TODO, it's not a marker, it's just a word.

```console
$ cat test.sh
#!/bin/sh

TODOS_DIR="$PWD"
$ pdd -v -f pdd.xml
My version is 0.23.1
Ruby version is 2.7.8 at x86_64-linux
Reading from root dir /data
/data/pdd.xml is a binary file (0 bytes)
Reading test.sh ...
ERROR: ERROR: test.sh; PDD::Error at test.sh:3: TODO found, but puzzle can't be parsed, most probably because TODO is not followed by a puzzle marker, as this page explains: https://github.com/cqfn/pdd#how-to-format
If you can't understand the cause of this issue or you don't know how to fix it, please submit a GitHub issue, we will try to help you: https://github.com/cqfn/pdd/issues. This tool is still in its beta version and we will appreciate your feedback. Here is where you can find more documentation: https://github.com/cqfn/pdd/blob/master/README.md.
Exit code is 1
```

**Comment by yegor256** [comment]
@pnatashap this is a reasonable bug report, maybe you can help us fix it?

**Comment by php-coder** [comment]
Fixed by #232
